docs/tutorial/spark-tutorial/t-setup.md

```diff
@@ -0,0 +1,141 @@
+## Setup Environment for the Tutorial
+
+This tutorial takes the demonstration on Charmed Spark by Paolo Sottovia at Ubuntu Summit 2023 as its reference. The video demonstration for this tutorial can be viewed on YouTube via [this link](https://www.youtube.com/watch?v=nu1ll7VRqbI).
```

Comment:
I totally agree

Reply:
Noted. I've even moved this part all the way to the end (further references section) because I think it would suit there better. Please let me know if you think otherwise.